tests/setup/profile_demo/doc_test.go

```diff
@@ -37,5 +37,11 @@ func TestDocs(t *testing.T) {
 }
 
 func setupConfig(ctx resource.Context, cfg *istio.Config) {
-	cfg.ControlPlaneValues = "profile: demo"
+	cfg.ControlPlaneValues = `
+profile: demo
+values:
+  pilot:
+    env:
+      PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING: true
```

Comment:
Should we do the same for profile_none?

Reply:
Profile none doesn't install istio, so nothing to add there. We would have to add it to all the tests when they install it to all tests that install istio if the tests use `istioctl x wait`.

Reply:
I did a quick search and the following tests look to be using _wait_for_istio_ and profile=none:
- observability/gateways
- observability/metrics/customize-metrics
- ops/config/traffic-management/network-topologies
- setup/getting-started
- traffic-management/egress/wildcard-egress
- traffic-management/egress/egress-control

Reply:
Seems kind of error prone to do this manually. I wonder if we can come up with a way for the framework to handle it automatically (e.g., change the default value of the config parameter)?